pkg/osquery/tables/execparsers/repcli/parser.go

```diff
@@ -0,0 +1,171 @@
+package repcli
+
+// repcli is responsible for parsing the output of the CarbonBlack
+// repcli sensor status utility. Some of the output format has
+// changed from the published documentation, as noted here:
+// https://community.carbonblack.com/t5/Knowledge-Base/Endpoint-Standard-How-to-Verify-Sensor-Status-With-RepCLI/ta-p/62524
+
+import (
+	"bufio"
+	"io"
+	"strings"
+	"unicode"
+)
+
+func NewStatusTemplate() map[string]map[string]any {
+	return map[string]map[string]interface{}{
+		"general_info": {
+			"devicehash":         "",
+			"deviceid":           "",
+			"quarantine":         "",
+			"sensor_version":     "",
+			"sensor_restarts":    "",
+			"kernel_type":        "",
+			"system_extension":   "",
+			"kernel_file_filter": "",
+			"background_scan":    "",
+			"last_reset":         "",
+			"fips_mode":          "",
+		},
+		"full_disk_access_configurations": {
+			"repmgr":           "",
+			"system_extension": "",
+			"osquery":          "",
+			"uninstall_helper": "",
+			"uninstall_ui":     "",
+		},
+		"sensor_status": {
+			"status":                      "",
+			"details":                     make(map[string][]string, 0),
+			"svcstable":                   "",
+			"boot_count":                  "",
+			"first_boot_after_os_upgrade": "",
+			"service_uptime":              "",
+			"service_waketime":            "",
+		},
+		"cloud_status": {
+			"registered":         "",
+			"server_address":     "",
+			"next_check-in":      "",
+			"private_logging":    "",
+			"next_cloud_upgrade": "",
+			"mdm_device_id":      "",
+			"platform_type":      "",
+			"proxy":              "",
+		},
+		"proxy_settings": {
+			"proxy_configured": "",
+		},
+		"enforcement_status": {
+			"execution_blocks":     "",
+			"network_restrictions": "",
+		},
+		"rules_status": {
+			"policy_name":               "",
+			"policy_timestamp":          "",
+			"endpoint_standard_product": "",
+			"enterprise_edr_product":    "",
+			"active_policies":           make(map[string]string, 0),
+		},
+	}
+}
+
+// formatKey prepares raw (potentially multi-word) key values by:
+// - stripping all surrounding whitespace
+// - coercing the entire string to lowercase
+// - splitting multiple words and joining them as snake_case
+func formatKey(key string) string {
+	processed := strings.TrimSpace(strings.ToLower(key))
+	words := strings.Fields(processed)
+	// State and Status are used interchangeably in section headers
+	// across device types, so we coerce all to status here
```

Comment:
How confident are you about that? Because this sort of thing means that if they evolve different meaning, we've backed ourselves into a corner and have bad data. I have a strong bias to presenting the information as it is.

Reply:
I was torn with this and would love more ideas here- the largest offender was that the linux output has a section header named `Sensor Status` and the same on mac was `Sensor State`. I think if we do what the rest of your comments are getting at (regarding flexibility) it would be less cumbersome to just let them be separate keys on separate distros in general

Reply:
Oh that's a good question... Hrm. Somewhat, more data needed.

I don't know if they'll be a time when `Sensor Status` and `Sensor State` mean different things. Or if a platform has both. Etc. I can say that in SQL, if we know about the differences we can say `select * where key in ('sensor', 'status')`, so it's easy to handle at that layer.

Reply:
that makes sense to me! I've updated to remove the coercion